src/index.ts

```diff
@@ -58,6 +58,16 @@ export function style(...objects: NestedCSSProperties[]) {
 }
 
 /**
+ * Takes CSSProperties and registers it to a global selector (body, html, etc.)
+ */
+export function rule(selector: string, ...objects: NestedCSSProperties[]): void {
```

Comment:
Maybe just `CSSProperties` as `&` interpolation and `@media` isn't actually supported by FreeStyle.registerRule.